src/centos/7/rpmpkg/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
+
+# Install the rpm packaging toolchain we need to build rpms
```

Comment:
Nit - consider avoiding `we` in the comment - maybe something like `Install the rpm packaging toolchain required to build rpms

Reply:
Fixed.